corehq/apps/api/resources/v0_5.py

```diff
@@ -1054,10 +1054,28 @@ def prepend_urls(self):
 
 
 class NavigationEventAuditResource():
+    LIMIT_DEFAULT = 10000
+
+    @classmethod
+    def cursor_query(cls, domain: str, local_time_zone: ZoneInfo,
```

Comment:
I'd never heard of cursor pagination, but I like it. What's the reason for having both the `query` below and this one? Or is the idea that `cursor_query` will always be the one to call `query`?

Reply:
No particular reason other than for function responsibility separation. `query` is for getting all the results we're interested in, and `cursor_query` is for paginating the results correctly. 

In the use case here, `query` is only called by `cursor_query ` but doesn't necessarily need to be the case.